js/src/pages/price-benchmark/index.scss

```
@@ -0,0 +1,3 @@
+.gla-price-benchmark__card {
+	padding: 0 32px;
```

@ankitrox Let's use the SCSS variables that we have available from: https://github.com/WordPress/gutenberg/blob/a4a5b07b4432d28ad4eb3646435bbaf9a8f9796e/packages/base-styles/_variables.scss#L58

```suggestion
	padding: 0 $grid-unit-40;
```